src/main/java/net/mcreator/element/types/Item.java

```diff
@@ -161,4 +178,54 @@ public boolean hasNonDefaultAnimation() {
 	public boolean hasEatResultItem() {
 		return isFood && eatResultItem != null && !eatResultItem.isEmpty();
 	}
+
+	public LinkedHashMap<String, Procedure> filterProperties() {

```

Comment:
Another thing I just saw.

If I see correctly, this "data accessor" is modifying the GE data model in the FTL generation process by calling

`customProperties.remove(property.getName())`

which is not ok. You can't modify data model while using it, is is antipattern. It would also help to add JavaDOC as it is not really clear what this method does, why it acts as a getter but has method name indicating changes to the data model.

Reply:
Turns out this method is useless as the map processed in it never contains vanilla properties and custom ones are by definition supported in any workspace.